src/Compilers/CSharp/Portable/BoundTree/Constructors.cs

```diff
@@ -256,14 +256,11 @@ public static BoundConversion SynthesizedNonUserDefined(SyntaxNode syntax, Bound
                 isBaseConversion: false,
                 @checked: @checked,
                 explicitCastInCode: explicitCastInCode,
-                isExplicitlyNullable: isExplicitlyNullable,
                 constantValueOpt: constantValueOpt,
+                conversionGroup,
                 type: type,
                 hasErrors: hasErrors || !conversion.IsValid)
         {
-            // PROTOTYPE(NullableReferenceTypes): Move assert to Validate method
-            // (add HasValidate="true" in BoundNodes.xml).
```

Comment:
>HasValidate="true" [](start = 20, length = 18)

Why remove the PROTOTYPE marker? Would such a `Validate` method still be useful?

Reply:
It might be useful to have a `Validate` method. Currently though, there's no requirement that `conversionGroupOpt` be set in lowering so it's not clear what we could assert.

---
In reply to: [189122524](https://github.com/dotnet/roslyn/pull/26356#discussion_r189122524) [](ancestors = 189122524)